<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableInterestDateWizardPage</class>
 <widget class="QWizardPage" name="VariableInterestDateWizardPage">
  <attribute name="title">
    <string>Date of next interest change</string>
  </attribute>
  <layout class="QVBoxLayout">
    <item>
      <widget class="QLabel" name="TextLabel8_2">
        <property name="text">
          <string>
Select the date when the interest rate for this loan will be modified and the frequency of the future changes.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer39">
        <property name="sizeHint">
          <size>
            <width>16</width>
            <height>99</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QGridLayout">
        <item row="0" column="0">
          <widget class="QLabel" name="TextLabel1">
            <property name="text">
              <string>Next interest change on</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item row="1" column="1">
          <layout class="QHBoxLayout">
            <item>
              <widget class="QSpinBox" name="m_interestFrequencyAmountEdit">
                <property name="minimum">
                  <number>0</number>
                </property>
                <property name="maximum">
                  <number>1000</number>
                </property>
              </widget>
            </item>
            <item>
              <widget class="KMyMoneyGeneralCombo" name="m_interestFrequencyUnitEdit">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item row="0" column="1">
          <widget class="KMyMoneyDateEdit" name="m_interestChangeDateEdit"/>
        </item>
        <item row="1" column="0">
          <widget class="QLabel" name="TextLabel2">
            <property name="text">
              <string>Time until next change</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer40">
        <property name="sizeHint">
          <size>
            <width>20</width>
            <height>80</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyDateEdit</class>
   <extends>KDateComboBox</extends>
   <header>kmymoneydateedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyGeneralCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneygeneralcombo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
